csrc/scheduler/matmul.cpp

```diff
@@ -181,6 +181,29 @@ void scheduleMatmul(
   // [... M,N,K]
   scheduler_utils::matmul_utils::makeTile(cc, gemm_tile.cta_tile.toVector());
 
+  // Applies swizzle factor on C
+  if (params.grid_swizzle_factor != 1) {
+    int factor = std::max(1, params.grid_swizzle_factor); // must be >=1
```

Comment:
We don't consider this type of tiling to be a swizzle. We consider swizzles to be non affine transformations. Can we just get a different word for this optimization? Tile shuffle or something similar might be a good name.

Reply:
We are already using the word `swizzle` for both affine and non-affine transformations. See note `[WarpMmaSwizzler]`.

Reply:
I went with CUTLASS's naming. What about grid scaling ? Or I'm fine with tile shuffle. So should I update @zasdfgbnm @csarofeen ?

Reply:
No it's fine then, we should probably have a non-affine vs affine swizzle rename, but doesn't seem to make a difference right now.

Reply:
Let's merge this PR as is so it is consistent with tracking-matmul.